aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -30,7 +30,7 @@ Follow the guidance in the <xref:blazor/get-started> article to create a Blazor
 
    The UI of the Counter component is defined using HTML. Dynamic rendering logic (for example, loops, conditionals, expressions) is added using an embedded C# syntax called [Razor](xref:mvc/views/razor). The HTML markup and C# rendering logic are converted into a component class at build time. The name of the generated .NET class matches the file name.
 
-   Members of the component class are defined in an `@functions` block. In the `@functions` block, component state (properties, fields) and methods are specified for event handling or for defining other component logic. These members are then used as part of the component's rendering logic and for handling events.
+   Members of the component class are defined in an `@code` block. In the `@code` block, component state (properties, fields) and methods are specified for event handling or for defining other component logic. These members are then used as part of the component's rendering logic and for handling events.
```

Comment:
We should probably also mention that `@code` and `@functions` are aliases. People might be confused if they already know about `@functions`.

Reply:
It is already mentioned in `components.md`